ionoscloud/resource_volume.go

```diff
@@ -650,6 +655,31 @@ func getVolumeData(d *schema.ResourceData, path string) (*ionoscloud.VolumePrope
 		if err := d.Set("ssh_key_path", [][]string{}); err != nil {
 			return nil, err
 		}
+		//if err := d.Set("ssh_keys", [][]string{}); err != nil {
+		//	return nil, err
+		//}
+	}
+	//else if v, ok := d.GetOk("ssh_keys"); ok { AR VENI PRIMUL ELSE IF MAI SUS
+	//	sshkeyPath = v.([]interface{})
+	//	if err := d.Set("ssh_key_path", v.([]interface{})); err != nil {
+	//		return nil, err
+	//	}
+	//}
+
+	if v, ok := d.GetOk(path + "ssh_keys"); ok {
+		sshkeyPath = v.([]interface{})
```

Comment:
@gabisavu-ionos again, here we are referring to `ssh_keys`, so please change the name of the variable. Please make this change in every place in this PR where we are referring to `ssh_keys`.